src/models/domain/cipherString.ts

```diff
@@ -105,6 +105,7 @@ export class CipherString {
         }
 
         try {
+            const orgKey = await cryptoService.getOrgKey(orgId);
```

Comment:
Why is this const declared and method called? Should it throw an exception OR throw away the result?

Reply:
I suspect this might have been the result of a merge conflict, I know another PR added support for overwriting the key to some models.